scripts/src/view/input/fugaziInput.tsx

```diff
@@ -116,21 +117,37 @@ export class FugaziInputView extends base.SuggestibleInputView<FugaziInputProper
 	}
 
 	protected onSuggestionItemPressed(item: statements.Statement): void {
-		let newValue: string = "",
+		let oldValueWords: string[] = this.getValue().split(" "),
+			expressionsIterator = (item.getExpression() as input.CommandExpression).getExpressions().getIterator(),
+			newValue: string = "",
 			haveStoppedForParameter = false,
 			iterator = item.getRule().getTokens().getIterator();
+
 		while (iterator.hasNext()) {
 			let token: syntax.RuleToken = iterator.next();
+			// by taking the value from the token we fix keyword typos
 			if (token.getTokenType() == syntax.TokenType.Keyword) {
```

Comment:
Probably better to use `===` instead of `==`
(https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons)